game/scripts/npc/abilities/leshrac_pulse_nova.txt

```diff
@@ -38,13 +38,13 @@
       }
       "damage" //OAA
       {
-        "value"                                           "100 150 200 350 500"
-        "special_bonus_unique_leshrac_3"                  "+60"
+        "value"                                           "90 140 190 340 490"
+        "special_bonus_unique_leshrac_3"                  "+50"
```

Comment:
You don't have to nerf the talent too. Pulse Nova dmg is already pathetic. Also it's a level 20 talent and every sane player will take the damage reduction talent over this.